grammars.yml

```
@@ -966,6 +971,15 @@ vendor/grammars/vscode-hack:
 vendor/grammars/vscode-hy:
 - source.hy
 vendor/grammars/vscode-ibmi-languages:
+- source.bnd
+- source.cl
+- source.dds.dspf
+- source.dds.icff
+- source.dds.lf
+- source.dds.pf
+- source.dds.prtf
+- source.mi
+- source.rpg
 - source.rpgle
```

See above. This list should only contain `source.rpgle`:

```suggestion
- source.rpgle
```